samples/LocalizationSample/Startup.cs

```diff
@@ -49,7 +52,7 @@ public void Configure(IApplicationBuilder app, IStringLocalizer<Startup> SR)
                     new CultureInfo("fr-FR"),
                     new CultureInfo("zh"),
                     new CultureInfo("zh-CN"),
-                    new CultureInfo("zh-CHT")
+                    //new CultureInfo("zh-CHT")
```

Comment:
This surprises me, given that full framework on Windows 10 will let anything through. Have you tried on non-Windows?